packages/build/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { buildSite } from './core/main.js'
 export { NetlifyPluginConstants } from './core/constants.js'
 
+export type { BufferedLogs as Logs } from './log/logger.js'
```

Comment:
Why export `BufferedLogs` as `Logs` specifically and not just `Logs` (that could be either `BufferedLogs` or `StreamedLogs`)? Especially as there are additions being made to `StreamedLogs` specifically so I imagine intention is the for called to use those?